components/activity/editor/collection/d2l-activity-editor-secondary-card.js

```diff
@@ -53,7 +56,7 @@ class ActivityEditorSecondaryCard extends LitElement {
 		<d2l-labs-accordion-collapse flex>
 			<span slot="header">
 				<h3>${this.titleText}</h3>
-				<hr>
+				<hr class="split">
```

Comment:
```suggestion
				<hr class="d2l-split">
```
Custom classes like this are always prefixed with `d2l-` by convention. Should probably change the other ones too.

Reply:
It has been done.